services/web/server/src/simcore_service_webserver/projects/models.py

```diff
@@ -22,6 +29,43 @@ def to_project_type_db(cls, api_type: "ProjectTypeAPI") -> ProjectType | None:
         }[api_type]
 
 
+class ProjectDB(BaseModel):
+    id: str
+    type: ProjectType
+    uuid: ProjectID
+    name: str
+    description: str | None
+    thumbnail: HttpUrlWithCustomMinLength | None
+    prj_owner: int
```

Comment:
Please double check, but make it a UserID or a GroupID (I think that is the former), cause that is going to bring a lot of confusion